src/document/printer.js

```diff
@@ -648,8 +680,8 @@ function printDocToString(doc, options) {
 
     return {
       formatted: beforeCursor + aroundCursor + afterCursor,
-      cursorNodeStart: beforeCursor.length,
-      cursorNodeText: aroundCursor,
+      cursorRegionStart: beforeCursor.length,
+      cursorRegionText: aroundCursor,
```

Comment:
`printDocToString` is a public API, let's not change the signature.

Reply:
Trouble is, the region we return here is no longer guaranteed to correspond to a single node. I guess I can leave the identifiers the same and add comments noting that they're misnomers if we want to preserve the signature, though.

Reply:
(Done)